tests/ui/type/type-ascription-precedence.stderr

```diff
@@ -9,12 +9,46 @@ error: expected identifier, found `:`
    |
 LL |     *(S: Z);
    |        ^ expected identifier
+   |
+help: if this is a parameter, remove the name for the parameter
+   |
+LL -     *(S: Z);
+LL +     *(Z);
+   |
+<<<<<<< HEAD
+=======
```

Comment:
Merge conflict?

Reply:
Yes, I messed up 😭. I'm so sorry I have no experience with Git.

Reply:
I'll fix this and implement the above change asap

Reply:
@nouritsu don't worry, that's ok. https://rustc-dev-guide.rust-lang.org/git.html#troubleshooting-git-issues has some help for some common cases, but as @compiler-errors mentioned, rebasing to a single commit before rebasing on top of master makes it easier to handle when you have to do it more than once.

Reply:
I understand, in the future whenever I contribute to the Rust compiler I'll use the following -
```
git add .
git commit --ammend -m "new changed + old ones"
```